validator/client/beacon-api/beacon_api_beacon_chain_client.go

```diff
@@ -22,16 +22,20 @@ import (
 
 type beaconApiBeaconChainClient struct {
 	fallbackClient          iface.BeaconChainClient
-	jsonRestHandler         jsonRestHandler
+	jsonRestHandler         JsonRestHandler
 	stateValidatorsProvider stateValidatorsProvider
 }
 
 const getValidatorPerformanceEndpoint = "/prysm/validators/performance"
 
 func (c beaconApiBeaconChainClient) getHeadBlockHeaders(ctx context.Context) (*beacon.GetBlockHeaderResponse, error) {
 	blockHeader := beacon.GetBlockHeaderResponse{}
-	if _, err := c.jsonRestHandler.GetRestJsonResponse(ctx, "/eth/v1/beacon/headers/head", &blockHeader); err != nil {
-		return nil, errors.Wrap(err, "failed to get head block header")
+	errJson, err := c.jsonRestHandler.Get(ctx, "/eth/v1/beacon/headers/head", &blockHeader)
```

Comment:
Since some endpoints are already constants should we add this one as well?

Reply:
I would prefer to have a separate task where we move all endpoints to globally available consts